subs/pantry/src/Pantry/Storage.hs

```diff
@@ -700,7 +732,7 @@ loadPackageById rpli tid = do
         Package
             { packageTreeKey = P.TreeKey blobKey
             , packageTree = mtree
-            , packageCabalEntry = pentry
+            , packageCabalEntry = pantry
```

Comment:
`pentry` was intended: it was the `package entry`.

Reply:
Lol. I did not realize that. I can see it now though. Can I rename it to `pEntry` or `packageEntry`, so I doesn't look like misspelled pantry? :)

Reply:
Either of those spellings is great.

Reply:
Switched to `packageEntry`